doc/theory/glossary.tex

```diff
@@ -263,10 +266,10 @@
 	||\lambda||_p = \left( \sum_{i \in \mathbb N} ||\lambda_i||^p_p \right)^{1/p}
 	\end{equation*}
 	whenever the right hand side exists and is finite.
-	
+
 	\paragraph{\\ References:} \cite{stein2011functional, bubenik2015statistical}
 	
-	\subsection*{Weighted silhouettes:} \label{weighted silhouettes}
+	\subsection*{Weighted silhouette:} \label{weighted silhouettes}
```

Comment:
Why the singular? I personally preferred the plural.

Reply:
I wanted to match other titles, like `persistence landscape`, `betti curve` etc

Reply:
I'm convinced, thanks!